.github/workflows/ci.yaml

```diff
@@ -18,3 +19,56 @@ jobs:
       run: bash <(curl -s https://codecov.io/bash)
       env:
         CODECOV_TOKEN: 4e256ab5-d474-459d-8d4d-cffea6455db7
+  release:
+    needs: test
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name != 'pull_request' || ! github.event.pull_request.head.repo.fork }}
+    steps:
+      - id: checkout
+        name: Checkout code
+        uses: actions/checkout@v2
+      - name: Login to GitHub Package Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
```

Comment:
I chose to use ghcr here but this can be replaced with any repository of choice. ghcr seemed easiest since it requires no secret setup. For other container repositories we will need to add an appropriate token to login to that registry.